tests/generic_config_updater/test_dhcp_relay.py

```diff
@@ -108,16 +110,14 @@ def default_setup(duthost, vlan_intfs_list):
     # Generate 4 dhcp servers for each new created vlan
     for vlan in vlan_intfs_list:
         expected_content_dict[vlan] = []
+        cmds.append('sonic-db-cli CONFIG_DB hset "VLAN|Vlan{}" "dhcp_servers@" "{}"'
```

Comment:
Hi Yaqiang, I don't think it is a good way to use sonic-db-cli to update the configDB. Can we use sudo config cli to add server at one time, like you presented earlier?

Reply:
Cli can only add multiple dhcp servers for single Vlan, there are more than one Vlan, config Cli has chances to hit same issue